Makefile

```diff
@@ -20,9 +20,47 @@ GOLANGCI_LINT = $(BIN_DIR)/golangci-lint
 # in case of a version bump
 GOLANGCI_LINT_VER = v1.63.4
 
+# Output directory
+BIN_DIR := bin
```

Comment:
u dont need this
we have it here:https://github.com/openshift-kni/commatrix/blob/45ce5346dde246bc49474cbcba18cdd9f1901fdf/Makefile#L14
this is why ur lint is failing